websites/T/TETR.IO/dist/metadata.json

```diff
@@ -9,13 +9,14 @@
   "description": {
     "en": "TETR.IO is a free-to-play online multiplayer and single player fan game"
   },
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
Why bumped to 1.1.0? You could just bump it to 1.0.2 because it's a minor fix...